metadata-ingestion/src/datahub/utilities/source_helpers.py

```diff
@@ -9,6 +9,16 @@
 from datahub.utilities.urns.urn import guess_entity_type
 
 
+def auto_workunit(
```

Comment:
Do we use this anywhere other than the test?

Reply:
not right now, but i'm planning on starting to use it elsewhere so I figured we should add it